dashboard/src/utils/addDay.js

```diff
@@ -0,0 +1,9 @@
+const addOneDay = (date) => {
```

Comment:
It would be nice if we could come up with a better name for this function.  That is, I don't think we want it to actually modify the input, and it returns a string, not a `Date` object.  So, perhaps something like `nextDayString` or `nextDateString`?